pydocx/tests/__init__.py

```diff
@@ -160,11 +158,15 @@ class WordprocessingDocumentFactory(object):
         </Types>
     '''  # noqa
 
-    def __init__(self, items=None):
+    def __init__(self, items=None, xml_header=None):
         self.items = items
         if not self.items:
             self.items = {}
 
+        if xml_header is None:
+            xml_header = '<?xml version="1.0" encoding="UTF-8"?>'
```

Comment:
If you're overriding the default, why not just make this string the default instead of None?